{{cookiecutter.project_name}}/.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,71 @@
+labels: bug
+body:
+    - type: markdown
+      attributes:
+          value: |
+              **Note**: Please read [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports)
+              detailing how to provide the necessary information for us to reproduce your bug. In brief:
+                * Please provide exact steps how to reproduce the bug in a clean Python environment.
+                * In case it's not clear what's causing this bug, please provide the data or the data generation procecure.
+                * Sometimes it is not possible to share the data but usually it is possible to replicate problems on publicly
+                  available datasets or to share a subset of your data.
+
+    - type: textarea
+      id: description
+      attributes:
+          label: Description of the bug
+          description: A clear and concise description of what the bug is.
+      validations:
+          required: true
+
+    - type: textarea
+      id: repex
+      attributes:
+          label: Minimal reproducible example
+          description: A code snipped that we can copy&paste to reproduce the bug without having any data
+          render: python
+          placeholder: |
+              import scanpy as sc
+              adata = sc.datasets.pbmc3k()
+
+              sc.do_something_that_doesnt_work(adata)
+
+    - type: textarea
+      id: stacktrace
+      attributes:
+          label: The error message produced by the code above
+          description: Please paste the entire error message
+          render: pytb
+          placeholder: |
+              ---------------------------------------------------------------------------
+              AssertionError                            Traceback (most recent call last)
+              <ipython-input-5-78750baef17a> in <module>
+              ----> 1 adata = sc.do_something_that_doesnt_work(adata)
+
+              /opt/conda/lib/python3.8/site-packages/scanpy/_preprocessing.py in do_something_that_doesnt_work(adata)
+                  438
+              --> 439     raise AssertionError("Demo function that fails")
+                  440
+
+    - type: textarea
+      id: versions
+      attributes:
+          label: Version information
+          description: |
+              Please provide use with some details about your system:
+
+               * Version of {{ cookiecutter.package_name }} (`{{ cookiecutter.package_name }}.__version__`)
+               * Version of numpy (`numpy.__version__`)
+               * Version of pandas (`pandas.__version__`)
+               * Version of AnnData (`anndata.__version__`)
+               * Python version (*e.g. 3.7, 3.8*)
+               * OS (*e.g. Arch Linux, Windows 10, MacOS*)
+          placeholder: |
+              {{ cookiecutter.package_name }}: 0.7.0
+              numpy: 1.22.3
+              pandas: 1.4.2
+              AnnData: 0.7.6
+              Python: 3.7
+              OS: Arch Linux
```

Comment:
@Zethson proposed of asking to use `session_info` here. The question is then if `session_info` should become a requirement in `pyproject.toml`

Reply:
Also not sure. 
1. session_info is not a dependency that is required to use the package
2. It is easier for newbies if they only install the package and then can use session_info instead of awkwardly having to install this as well.

I would lean towards making it a dependency. What do the others think?

Reply:
In principle, I agree with making it a dependency, but there's also https://github.com/scverse/cookiecutter-scverse/issues/57#issuecomment-1136161434

@scverse/core-devs ?

Reply:
I'd just make it a dependency for now.